src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -1885,6 +2014,12 @@ size_t Eltwise::getOpInputsNum() const {
         case Algorithm::EltwiseMulAdd:
         case Algorithm::EltwiseSelect:
             return 3;
+        case Algorithm::EltwiseBitwiseAnd:
```

Comment:
Perhaps it is better to keep the division into groups and move it to the line #2013?

Reply:
done, thanks